grammar.js

```diff
@@ -177,7 +177,7 @@ module.exports = grammar({
     $._external_close_bracket2
   ],
 
-  word: $ => $._identifier,
+  word: $ => $.identifier,
```

Comment:
Note `_identifier` -> `identifier`, which just includes quoted identifiers in the word check now, but I don't think that is going to matter much after reading this again https://tree-sitter.github.io/tree-sitter/creating-parsers#keyword-extraction